src/Faker/Provider/Address.php

```diff
@@ -104,20 +104,24 @@ public static function country()
     }
 
     /**
-     * @example 77.147489
+     * @example '77.147489'
+     * @param float|int $min
+     * @param float|int $max
      * @return float Uses signed degrees format (returns a float number between -90 and 90)
      */
-    public static function latitude()
+    public static function latitude($min = -90, $max = 90)
```

Comment:
You're changing the results of existing calls. Please use 0 and 180 as min and max values.